src/jl_exported_data.inc

```diff
@@ -54,6 +54,7 @@
     XX(jl_int8_type) \
     XX(jl_interconditional_type) \
     XX(jl_interrupt_exception) \
+    XX(jl_interrupt_handlers) \
```

Comment:
```suggestion
```

Reply:
We're exporting this and using `cglobal` to access it instead of accessor functions